### ISSUE TITLE
[BUG] OSB does not gracefully handle being invoked with no arguments

### ISSUE THREAD

**Issue by JeffH-AWS** [opensearch-project/opensearch-benchmark#237]
**Describe the bug**
As a new OpenSearch Benchmark user, when I try to run the command to learn how to use it, I'm presented with a traceback:

```bash
$ opensearch-benchmark
Traceback (most recent call last):
  File "/home/ec2-user/.local/bin/opensearch-benchmark", line 8, in <module>
    sys.exit(main())
  File "/home/ec2-user/.local/lib/python3.9/site-packages/osbenchmark/benchmark.py", line 949, in main
    console.init(quiet=args.quiet)
AttributeError: 'Namespace' object has no attribute 'quiet'
```

**To Reproduce**
Install OpenSearch Benchmark and run the command `opensearch-benchmark` with no options.

**Expected behavior**
I would expect OpenSearch Benchmark to handle a lack of arguments by presenting me with the same output as the `-h/--help` option. Like this:

```bash
$ opensearch-benchmark -h
usage: opensearch-benchmark [-h] [--version] {execute_test,list,info,create-workload,generate,compare,download,install,start,stop} ...

   ____                  _____                      __       ____                  __                         __
  / __ \____  ___  ____ / ___/___  ____ ___________/ /_     / __ )___  ____  _____/ /_  ____ ___  ____ ______/ /__
 / / / / __ \/ _ \/ __ \\__ \/ _ \/ __ `/ ___/ ___/ __ \   / __  / _ \/ __ \/ ___/ __ \/ __ `__ \/ __ `/ ___/ //_/
/ /_/ / /_/ /  __/ / / /__/ /  __/ /_/ / /  / /__/ / / /  / /_/ /  __/ / / / /__/ / / / / / / / / /_/ / /  / ,<
\____/ .___/\___/_/ /_/____/\___/\__,_/_/   \___/_/ /_/  /_____/\___/_/ /_/\___/_/ /_/_/ /_/ /_/\__,_/_/  /_/|_|
    /_/

 A benchmarking tool for OpenSearch

optional arguments:
  -h, --help            show this help message and exit
  --version             show program's version number and exit

subcommands:
  {execute_test,list,info,create-workload,generate,compare,download,install,start,stop}
    execute_test        Run a benchmark
    list                List configuration options
    info                Show info about a workload
    create-workload     Create a Benchmark workload from existing data
    generate            Generate artifacts
    compare             Compare two test_executions
    download            Downloads an artifact
    install             Installs an OpenSearch node locally
    start               Starts an OpenSearch node locally
    stop                Stops an OpenSearch node locally

Find out more about Benchmark at https://opensearch.org/docs
```

**Logs**
N/A

**More Context (please complete the following information):**
- Workload N/A
- Service N/A
- Version `opensearch-benchmark 0.2.0`

**Additional context**
N/A

**Comment by IanHoang** [comment]
Thanks for bringing this up. Will take a look into this.

**Comment by IanHoang** [comment]
Closing issue as this has been closed in PR #247
